packager/hls/base/master_playlist_unittest.cc

```diff
@@ -246,12 +251,12 @@ TEST_F(MasterPlaylistTest, WriteMasterPlaylistVideoAndAudio) {
   // First audio, english.m3u8.
   std::unique_ptr<MockMediaPlaylist> english_playlist = CreateAudioPlaylist(
       "eng.m3u8", "english", "audiogroup", "audiocodec", "en", kAudio1Channels,
-      kAudio1MaxBitrate, kAudio1AvgBitrate);
+      kAudio1MaxBitrate, kAudio1AvgBitrate, false, 0);
```

Comment:
Can we define constants in line 35 to make the code more readable?

```c++
const kEC3JocFlagEnabled = true;
const kEC3JocComplexityZero = 0
const kEC3JocComplexity = 16;
```

Then we can use: 

```c++
kAudio1MaxBitrate, kAudio1AvgBitrate, !kEC3JocFlagEnabled, kEC3JocComplexityZero);
```

Reply:
Good suggestion. I will update it.